### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by geniusgordon**
typo virdect->verdict

### CODE REVIEW

**Comment by nickelcheng** [comment]
+1, @bruce3557 please check

**Comment by bruce3557** [comment]
+1, please merge latest code and I can merge it

**Comment by geniusgordon** [comment]
OK~

**Comment by nickelcheng** [comment]
going tot merge.

File: problem/models.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ class SubmissionDetail(models.Model):
     sid = models.ForeignKey(Submission)
```
Comment (henryyang42) [inline problem/models.py]: VIRDECT_CHOICE above is not fixed.

Comment (nickelcheng) [inline problem/models.py]: ok, thanks a lot.

**Pull request merged**